aiodebug/tests/test_log_slow_callbacks.py

```diff
@@ -27,16 +26,16 @@ def test_slow_callback_causes_a_warning(get_logger):
 
 
 @mock.patch('aiodebug.logging_compat.get_logger')
-def test_waiting_callback_does_not_cause_warning(get_logger):
+def test_waiting_callback_does_not_cause_warning(get_logger, event_loop):
 	logger = mock.Mock()
 	get_logger.return_value = logger
 
-	loop = asyncio.get_event_loop()
+	asyncio.set_event_loop(event_loop)
 	aiodebug.log_slow_callbacks.enable(0.02)
 
 	async def waitingcb():
 		await asyncio.sleep(0.05)
 
-	loop.run_until_complete(waitingcb())
+	event_loop.run_until_complete(waitingcb())
 
```

Comment:
Because the tests use common asyncio loop and interfere with each other. I had to create isolated event loops for some tests to keep them from failing.